packages/cli/src/commands/schema/update.ts

```diff
@@ -44,10 +51,13 @@ export default class SchemaUpdateCommand extends APIOrganizationCommand<typeof S
 
 		const getInputFromUser = async (): Promise<SchemaAppRequest> => {
 			const original = await this.client.schema.get(id)
+			if (original.certificationStatus === 'wwst' || original.certificationStatus === 'cst') {
+				this.cancel('Schema apps that have already been certified cannot be updated.')
```

Comment:
Consider "...cannot be updated via the CLI."

Reply:
Fixed